actions/copy.js

```diff
@@ -28,8 +29,8 @@ exports.copy = function(from, to, options) {
   );
 
   var globOptions = _.extend(options.globOptions || {}, { nodir: true });
-  var files = glob.sync(from, globOptions);
-  var root = util.getCommonPath(from);
+  var files = globby.sync(from, globOptions);
+  var root = util.getCommonPath(Array.isArray(from) ? files : from);
```

Comment:
I'd like to think so, but I didn't want to introduce a potentially breaking change.